pymilvus/milvus_client/defaults.py

```diff
@@ -0,0 +1,26 @@
+"""Default vectorstore args."""
+
+DEFAULT_COLLECTION_ARGS = {
+    "collection_name": "ClientStore",
+}
+DEFAULT_LOAD_ARGS = {
+    "partition": None,
+    "replica_number": 1,
+}
+
+DEFAULT_CONNECTION_ARGS = {
+    "alias": "default",
+}
+
+DEFAULT_SEARCH_PARAMS = {
```

Comment:
we should change all the default metric type to cosine in the future.

Reply:
Yes, for now I was putting L2 as we cant enforce that the data is normalized which will create very poor results. If we end up normalizing at the segment level it may be good enough to default to.